src/traces/parcoords/defaults.js

```diff
@@ -53,7 +59,18 @@ function dimensionDefaults(dimensionIn, dimensionOut) {
         coerce('tickvals');
         coerce('ticktext');
         coerce('tickformat');
-        coerce('range');
+
+        dimensionOut._ax = {
+            _id: 'y',
+            type: 'linear',
+            showexponent: 'all',
+            exponentformat: 'B'
+        };
+
+        Axes.setConvert(dimensionOut._ax, opts.layout);
+
+        axCoerce('range');
+        dimensionOut.range = dimensionOut._ax.range;
```

Comment:
I don't understand why you coercing `range` using the cartesian axis attributes and then cross-referencing the result back to `fullData[i].dimensions[j].range`

Can you instead:

- :hocho: that `axCoerce`
- call `coerce('range')` as before
- set `dimensionOut._ax.range` to `coerce('range')` result

Reply:
... by the way, calling `Axes.setConvert` during the defaults is a great idea :+1:

Reply:
@etpinard Thanks very much for the note. 
Done in https://github.com/plotly/plotly.js/pull/3966/commits/c8dedcb31c0043ce3d5433b29af1e1f8e430c38f.

Reply:
Thank you!